models/project_board.go

```diff
@@ -266,6 +266,7 @@ func (b *ProjectBoard) LoadIssues() (IssueList, error) {
 		issues, err := Issues(&IssuesOptions{
 			ProjectBoardID: b.ID,
 			ProjectID:      b.ProjectID,
+			SortType:       "project-sorting",
```

Comment:
Should be "board-sorting"? The sorting is for every board.

Reply:
In favor of #13802 I would name it `project-column-sorting` as `column-sorting` would be a bit misleading IMO.

Reply:
changed by cd3028a9f0810b6ca563e8582be572f4cc881aeb